<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="editUser">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: white;   /* fond blanc */
    color: black;              /* texte noir */
    border: 1px solid rgb(41,127,110);  /* contour vert */
    border-radius: 6px;        /* coins arrondis */
    padding: 4px 8px;          /* marges internes */
    font-size: 14px;
}

QLineEdit:focus {
    border: 2px solid rgb(41,127,110); /* contour plus épais quand actif */
    background-color: rgb(250, 250, 250); /* très léger gris pour focus */
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="editPass">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: white;   /* fond blanc */
    color: black;              /* texte noir */
    border: 1px solid rgb(41,127,110);  /* contour vert */
    border-radius: 6px;        /* coins arrondis */
    padding: 4px 8px;          /* marges internes */
    font-size: 14px;
}

QLineEdit:focus {
    border: 2px solid rgb(41,127,110); /* contour plus épais quand actif */
    background-color: rgb(250, 250, 250); /* très léger gris pour focus */
}
</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>90</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(209, 209, 209);
    color: rgb(41, 127, 110);
    border: none;
    border-radius: 5px;
    padding: 1px 20px;
    font-size: 14px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: rgb(41, 127, 110);
    color: white;
    border: 2px solid white;   /* ✅ visible white border on hover */
    border-radius: 8px;
    padding: 6px 18px;         /* ✅ keeps button size consistent */
}
</string>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>90</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(209, 209, 209);
    color: rgb(41, 127, 110);
    border: none;
    border-radius: 5px;
    padding: 1px 20px;
    font-size: 14px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: rgb(41, 127, 110);
    color: white;
    border: 2px solid white;   /* ✅ visible white border on hover */
    border-radius: 8px;
    padding: 6px 18px;         /* ✅ keeps button size consistent */
}
</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>19</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    /* normal state */
    color: rgb(41, 127, 110);
    border: none;
    border-radius: 8px;
    padding: 8px 20px;
    font-size: 15px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Nom de l'utilisateur :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    /* normal state */
    color: rgb(41, 127, 110);
    border: none;
    border-radius: 8px;
    padding: 8px 20px;
    font-size: 15px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Mot de passe :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
